app/views/internal/users/show.html.erb

```diff
@@ -36,65 +36,8 @@
       <dd><%= @user.monthly_dues %></dd>
     <% end %>
   </dl>
-
-  <p><strong><u>Activity</u></strong></p>
-  <ul>
-    <li><%= @user.comments.size %> comments</li>
-    <li><%= @user.articles.size %> articles</li>
-    <li><%= @user.reactions.size %> reactions</li>
-    <li><%= @user.followers.size %> followers</li>
-    <li><%= @user.following_users_count %> users following</li>
-    <li><%= @user.badge_achievements_count %> Badges</li>
-
-  </ul>
-
-</div>
-<div class="row">
-  <h3><u>Current Roles</u></h3>
-  <p>(view full history in notes below)</p>
-  <ul>
-    <% @user.roles.each do |role| %>
-      <li><%= role.name %> <em><%= "- " + Tag.find(role.resource_id).name if role.resource_id.present? %></em></li>
-    <% end %>
-  </ul>
-  <h3><u>Privileges</u></h3>
-  <% if !@user.banned && !@user.warned %>
-    <%= form_for [:internal, @user] do |f| %>
-      <p>
-        Add Trusted Role (Community Moderator)
-        <%= f.check_box :trusted_user, checked: @user.trusted %>
-      </p>
-      <p>
-        Add Pro Role (Upgraded DevRel Dashboard)
-        <%= f.check_box :pro_user, checked: @user.has_role?(:pro) %>
-      </p>
-      <%= f.submit "Update Privileges" %>
-    <% end %>
-  <% else %>
-    User must be in good standing to have heigtened privileges of any kind.
-  <% end %>
-  <br>
-  <h3><u>Warn or Ban User</u></h3>
-  <% if @user.banned %>
-    <p style="background-color:red;color:white">🚨 Banned User 🚨</p>
-  <% elsif @user.warned %>
-    <p style="background-color: orange;color:white">Warned User</p>
-  <% else %>
-    <p style="color:green">Member in good standing</p>
-  <% end %>
-  <%= form_for [:internal, @user] do |f| %>
-    <%= f.label "Ban User" %>
-    <%= f.check_box :ban_user, checked: @user.banned %>
-    <br>
-    <%= f.label "Warn User" %>
-    <%= f.check_box :warn_user, checked: @user.warned %>
-    <br>
-    <%= f.label "Reason for action:" %>
-    <%= f.text_area :note_for_current_role, required: true %>
-    <br>
-    <%= f.submit "Update User Status" %>
-  <% end %>
 </div>
+<%= render "internal/users/activity" %>
```

Comment:
Since `activity` is a partial, this should work right?
```suggestion
<%= render "activity" %>
```

Reply:
Yeah, as long as the scoping is appropriate. Sometimes we use partials from different scopes, but this should be fine.